tests/integration/blockchain/test_wallet_server_sessions.py

```diff
@@ -49,3 +51,36 @@ class TestSegwitServer(IntegrationTestCase):
 
     async def test_at_least_it_starts(self):
         await asyncio.wait_for(self.ledger.network.get_headers(0, 1), 1.0)
+
+
+class TestUsagePayment(CommandTestCase):
+    LEDGER = lbry.wallet
+
+    def setUp(self) -> None:
+        WalletServerPayer.PAYMENT_PERIOD = 1
```

Comment:
Can this just be a configurable thing in the WalletServerPayer (eg. an argument to WalletServerPayer.__init__ which then uses the PAYMENT_PERIOD constant as a default)?